pkgs/by-name/st/stripe-cli/package.nix

```diff
@@ -2,13 +2,13 @@
 
 buildGoModule rec {
   pname = "stripe-cli";
-  version = "1.21.10";
+  version = "1.22.0";
 
   src = fetchFromGitHub {
     owner = "stripe";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-L2ydkrm2foSIh67LsMYNvV53Hdj9TdV7C4tKeHe+AK0=";
+    hash = "sha256-h5Q9+WoMJ5swmyL4GIiWZ9HDnr/w23iBVtp4EQ1yYn0=";
   };
   vendorHash = "sha256-TuxYJ3u4/5PJYRoRgom+M1au9XerZ+vj9X3jUWTPM58=";
```

Comment:
I'm amazed the dependencies haven't been updated for 6 months...

Reply:
As am I 🙂

I took the liberty of adding a minor changes that removes the use of `pname` for the `repo` attribute.